lib/src/component/ref_util.dart

```diff
@@ -225,7 +225,7 @@ UiFactory<TProps> Function(UiFactory<TProps>) forwardRef<TProps extends UiProps>
     }
 
     ReactComponentFactoryProxy hoc = react_interop.forwardRef(wrapProps, displayName: displayName);
-    setComponentTypeMeta(hoc, isHoc: true, parentType: factory().componentFactory);
+    setComponentTypeMeta(hoc.type, isHoc: true, parentType: factory().componentFactory.type);
```

Comment:
Was this something you discovered was not working correctly - or is this necessary for these changes to work?

Reply:
It was working fine before; this was necessary since I changed `setComponentTypeMeta`, a private API, to expect the raw type instead of the proxy.